ts/Core/Chart/ChartDefaults.ts

```diff
@@ -426,6 +426,21 @@ const ChartDefaults: ChartOptions = {
      * @apioption chart.numberFormatter
      */
 
+    /**
+     * The number of times the loop repeats the calculation of axis future
+     * dimensions while rendering the chart to find perfect axis fills and tick
+     * intervals. Use this property if the axes change after you redraw the
+     * chart.
+     *
+     * **Note:** Changing that option to higher than the default might decrease
+     * performance significantly, especially with bigger sets of data.
+     *
+     * @type      {number}
+     * @default   1
+     * @since     @next
+     * @apioption chart.overheatProtector
```

Comment:
We need a more descriptive name. Like `chart.axisFittingRuns`? Any better idea?

Reply:
Some other ideas: `initFittingMaxIterations`, `axesLayoutLoopLimit`, `maxReflowTries`.

Reply:
What do you think about `axesLayoutRuns`?